lstchain/scripts/lstchain_data_muon_analysis_dl1.py

```diff
@@ -43,13 +48,19 @@
 parser.add_argument("--plots_path", help = "Path to the plots",
                     default = None, type = str)
 
+parser.add_argument("--max_muons", help = "Maximum number of processed muon ring candidates",
+                    default = -1, type = int)
+
 args = parser.parse_args()
 
 
 def main():
 
-    print("input file: {}".format(args.input_file))
+    print("input files: {}".format(args.input_file))
+    print("calib file: {}".format(args.calib_file))
     print("output file: {}".format(args.output_file))
+    
+    max_muons = args.max_muons
 
     # Camera geometry
     geom = CameraGeometry.from_name("LSTCam-002")
```

Comment:
geometry needs to be updated, could you do it here, please?

Reply:
`LSTCam-004`
from 002 to 003, there was a wrong camera rotation:
https://github.com/cta-observatory/ctapipe-extra/pull/25

from 003 to 004 there was only a small pixel rotation:
https://github.com/cta-observatory/ctapipe_io_lst/pull/23

Reply:
> `LSTCam-004`
> from 002 to 003, there was a wrong camera rotation:
> [cta-observatory/ctapipe-extra#25](https://github.com/cta-observatory/ctapipe-extra/pull/25)
> 
> from 003 to 004 there was only a small wrong pixel rotation:
> [cta-observatory/ctapipe_io_lst#23](https://github.com/cta-observatory/ctapipe_io_lst/pull/23)

Ok, done now